File: 프로그래머스/0/181874. A 강조하기/A 강조하기.cs
using System;

class Solution {
    public String solution(String myString) {
        
        char[] ch = new char[myString.Length];
        for(int i = 0; i < myString.Length;i++){
            ch[i]= myString[i];
            if((int)ch[i] == 97){
                ch[i] = 'A';
            }
            else if((int)ch[i] != 'A' && (int)ch[i] <91 && (int)ch[i] > 64){
                ch[i] = (char)(ch[i]+32);
            }
            else{
                
            }
        }
        String answer = new String(ch);
        return answer;
    }
}

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.js
function solution(myString) {
    var answer = '';
    for(let i = 0; i < myString.length; i++){
        if(myString[i] == "A" || myString[i] == "a"){
            answer += myString[i].toUpperCase();
        }else{
            answer += myString[i].toLowerCase();
        }
    }
    return answer;
}